### ISSUE TITLE
Add option to change browser when editing app

### ISSUE THREAD

**Issue by spsf64** [linuxmint/webapp-manager#224]
During creation of a new app we can choose the default browser, but later if you want to edit such app, there is no option to select the browser..

**Comment by haggen88** [comment]
I agree, at the moment the only way to change the browser is to generate the webapp from scratch.

**Comment by haggen88** [comment]
#131 related
